index.html

```diff
@@ -13,39 +13,12 @@
     <link rel="manifest" href="/site.webmanifest">
     <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
     <meta name="msapplication-TileColor" content="#da532c">
-    <meta name="theme-color" content="#ffffff">
-    
-    <style>
-       h1{ background-image:url("images/gradient.png");
-       font-family: 'Titillium Web', sans-serif;
-       }
-        
-       h2{ background-image:url("images/gradient.png");
-       font-family: 'Quicksand', sans-serif;
-       }
-       
-       #doom{ text-indent: 3em;
-       }
-       footer{ font-size: 12px;
-       }
-       .byline{font-size: 70%;
-       }
-       a {text-decoration: none;
-       }
-       #page-copyright {text-transform: uppercase;
-       letter-spacing: 3px;
-       }
-      .om
-      {list-style: none;
-       }
-       ul
-         {list-style: circle;}
-     </style>  
+    <meta name="theme-color" content="#ffffff"> 
 
 </head>
 <body>
     <header>
-        <nav class="om">
+        <nav>
```

Comment:
no need for this new class.  just use a descendent selector like `header nav ul`